app/src/main/java/com/sunasterisk/englishflashcard/data/repository/DictionaryRepository.kt

```diff
@@ -0,0 +1,20 @@
+package com.sunasterisk.englishflashcard.data.repository
+
+import com.sunasterisk.englishflashcard.data.model.Dictionary
+import com.sunasterisk.englishflashcard.data.source.DictionaryDataSource
+import com.sunasterisk.englishflashcard.data.source.local.DictionaryLocalDataSource
+
+class DictionaryRepository (private val localDataSource: DictionaryLocalDataSource) :
```

Comment:
```suggestion
class DictionaryRepository(
    private val localDataSource: DictionaryDataSource.Local
):
```

Triển khai theo Singleton Pattern đi em